bukuserver/util.py

```diff
@@ -0,0 +1,17 @@
+from collections import Counter
+from urllib.parse import urlparse
+import re
+
+get_netloc = lambda x: urlparse(x).netloc  # pylint: disable=no-member
+select_filters = lambda args: {k: v for k, v in args.items() if re.match(r'flt.*_', k)}
```

Comment:
A `|flt` Jinja filter to prevent copying non-filter params into the random entry URL.